docker/Dockerfile

```diff
@@ -85,4 +91,5 @@ RUN ./flatc --cpp -o /home/ubuntu/software/jasminegraph/src/util/dbutil /home/ub
 WORKDIR /home/ubuntu/software/jasminegraph
 ENV JASMINEGRAPH_HOME="/home/ubuntu/software/jasminegraph"
```

Comment:
Since this is docker environment we will be using a new Linux image. Within the image the folder `/home/ubuntu/software/jasminegraph` will be created. Hence we will not have to declare an environment variable. Hence resolving this conversation.